gremlin-test/src/main/resources/org/apache/tinkerpop/gremlin/test/features/sideEffect/Inject.feature

```diff
@@ -231,6 +250,19 @@ Feature: Step - inject()
       | result |
       | l[d[1].i,d[3].i,d[100].i,d[300].i] |
 
+  @GraphComputerVerificationInjectionNotSupported @GremlinLangScriptOnly
```

Comment:
what is `GremlinLangScriptOnly` for? if it's a new tag, then it should be documented in the developer docs.

Reply:
It's a tag to skip scenarios which are not supported in gremlin groovy, even with the help of the groovy translator. The only scenarios which require these skips in this branch which have parameterized nulls passed to ambiguous overloads.

I will update the provider docs and perhaps rename to `@GremlinGroovyNotSupported`